### PR TITLE
created product page and fetch data from  mockbee

### PR DESCRIPTION
**Pull request by Rockyraven**
here I have created product page refactored from react 
fetch product data from mockbee backened
product page refactored

### CODE REVIEW

File: src/assets/index.js [hunk 1]
```diff
@@ -2,5 +2,12 @@ import brandBanner from "./brand-banners.jpg"
 import menShoe from "./mens-shoe.jpg"
 import womenShoe from './women-shoe.jpg'
 import kidShoe from './kidshoe.jpg'
+import imgShoe1 from './shoe.jpg'
```
Comment (sonishreyas) [inline src/assets/index.js:5]: You can directly do export here, you don't need to use import

**Review by sonishreyas — APPROVED** [review]
The code looks good and easy to understand. I had one suggestion other than that code is good

File: src/assets/index.js (hunk 1 above)
Comment (Rockyraven) [reply src/assets/index.js:5]: from this way I comfortable
this reduce line of code and easy to understand that from where i am importing our code

File: src/component/filter/Filter.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+            <div class="shoetype-category">
+                    <p class="filter-subtitle">Categories</p>
+                    <ul>
+                        <li><input type="checkbox"/> <label for="input">Sports Shoes</label></li>
+                        <li><input type="checkbox"/> <label for="input">Formal Shoes</label></li>
+                        <li><input type="checkbox"/> <label for="input">Sneakers Shoes</label></li>
+                        <li><input type="checkbox"/> <label for="input">Casual Shoes</label></li>
+                        <li><input type="checkbox"/> <label for="input">Sandles & Floater</label></li>
+                    </ul>
```
Comment (nyyshaaa) [inline src/component/filter/Filter.jsx]: is it working fine with for,like shouldn't it be htmlFor

File: src/pages/product/Product.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    .then(data=>setProduct(data.products))
+
+  },[])
+  return (
+    <>
+    <div className="map">
+     <div> <Filter /></div>
+      <div>{product.map(item=><ProductCard title={item.title}  brand={item.brand} price={item.price} discount={item.discount} actualPrice={item.actualPrice} imgSrc={item.image} />) }</div>
+      </div>
+    </>
```
Comment (nyyshaaa) [inline src/pages/product/Product.js]: you can use products here ,although it's not necessary just to convey it's plural.

**Review by nyyshaaa — APPROVED** [review]
just some minor suggestions, all else looks good and clean.

**Pull request merged**